src/cascadia/TerminalApp/App.cpp

```diff
@@ -1547,8 +1547,12 @@ namespace winrt::TerminalApp::implementation
                 dataPack.SetHtmlFormat(htmlData);
             }
 
-            Clipboard::SetContent(dataPack);
-            Clipboard::Flush();
+            try
+            {
+                Clipboard::SetContent(dataPack);
+                Clipboard::Flush();
+            }
+            CATCH_LOG();

```

Comment:
Strange thing is that it still crashes to me (every time I do copy). However if I change `CRASH_LOG();` to `catch(...) { }` then there is no longer a crash.